.eslintrc.js

```diff
@@ -2,8 +2,19 @@
 
 const config = {
   root: true,
-  env: { commonjs: true, node: true, browser: true },
-  extends: ['ackama', 'ackama/react'],
+  env: {
+    commonjs: true,
+    node: true,
+    browser: true,
+    jest: true
+  },
```

Comment:
```suggestion
    browser: true
  },
```

`eslint-plugin-jest` enables this.